### ISSUE TITLE
tex2pdf.* directory persists after knit

### ISSUE THREAD

**Issue by BarkleyBG** [yihui/knitr#1291]
Sometimes (but not always) when I knit an Rmarkdown document, an empty folder called "tex2pdf.*" persists after knitting finishes.

Description of issue:
I have a document (e.g. test_knit.Rmd) in a directory (~/myKnitDir/) When I knit the .Rmd  with output .pdf, both of the following are briefly created: (1) myKnitDir/test_knit.knit.md (2)myKnitDir/tex2pdf.*/, which is a folder where the asterisk is a placeholder for what is usually 3-5 numerals. Usually both (1) and (2) are quickly and automatically removed from ~/myKnitDir/, but sometimes (2) continues to exist. When they are created, these directories include a file called input.tex, which is quickly and automatically deleted. The directories like (2) are empty when they persist. After several of these occurences, these empty directories pile up.

Description of occurence:
- it can happen the first time knitting a fresh .Rmd, but not necessarily.
- it can happen the n^th time knitting a fresh .Rmd, but not necessarily.
- it can happen when knitting the .Rmd without making any edits to the document since the last knit, but not necessarily
- it can happen when knitting the .Rmd after having made edits to the document since the last knit, but not necessarily
- it can happen when I mouse over from Rstudio into the Windows Explorer directory that the .Rmd file exists in (e.g. ~/myKnitDir) while the document is being knitted, but not necessarily.

Here is the sample code for test_knit.Rmd that can (sometimes) reproduce the issue.

    ---
    title: "knitr_testr"
    author: "Brian Barkley"
    date: "September 15, 2016"
    output: pdf_document
    ---
    
    \textbf{Yihui} asked me to test this!!
    
    ```{r}
    devtools::session_info('rmarkdown')
    ```
    
    Here goes

Attached is the output test_knit.pdf with session_info() shown.
[test_knit.pdf](https://github.com/yihui/knitr/files/475597/test_knit.pdf)

Thanks!

**Comment by yihui** [comment]
Before I jump in, is it too much trouble for you to upgrade your R version and all R packages? If you don't do that, I will have to go the opposite direction :(

**Comment by yihui** [comment]
I did more research on this issue, and I'm relatively sure this is a Pandoc issue or a Windows issue. It is probably the same as https://github.com/jgm/pandoc/issues/1192 (the OP there said he/she was in Dropbox) Neither knitr nor rmarkdown actually creates this tex2pdf.* directory. If this bothers you much, I can propose a patch to rmarkdown to remove this empty directory after the Pandoc conversion.

Closing this issue only because knitr is not really involved.
